media/css/externalpages/pocket/components/privacy-tos.scss

```diff
@@ -44,6 +45,10 @@
 
     p {
         color: $color-text-secondary;
+
+        .uppercase {
+            text-transform: uppercase;
+        }
```

Comment:
Should have mentioned this in the `privacy` page PR, but let's also add a `&:focus, &:active` to the anchor tag, that way when people are tabbing through the page, they will also see the color change and whatnot